test/functional/services/common/test_subjects.ts

```diff
@@ -361,10 +361,14 @@ export class TestSubjects extends FtrService {
   }
 
   public async waitForEnabled(selector: string, timeout: number = this.TRY_TIME): Promise<void> {
-    await this.retry.tryForTime(timeout, async () => {
+    this.log.debug(`TestSubjects.waitForEnabled(${selector}) with timeout=${timeout}`);
+    const success = await this.retry.tryForTime(timeout, async () => {
       const element = await this.find(selector);
       return (await element.isDisplayed()) && (await element.isEnabled());
     });
+    if (!success) {
+      throw new Error(`The element ${selector} was not enabled within the ${timeout}ms timeout.`);
+    }
```

Comment:
`waitForEnabled` didn't actually cause a test failure, so the tests continued despite the item I was waiting for **never being enabled** - I think it's more helpful if this method **actually fails** in this case rather than simply continuing, so I added it.